docs/src/main/sphinx/sql/select.md

```diff
@@ -4,6 +4,7 @@
 
 ```text
 [ WITH FUNCTION sql_routines ]
+[ WITH SESSION name = expression [, ...] ]
```

Comment:
This is incorrect per the proposed grammar. The combined form when inline functions are present would be:

```
WITH 
  FUNCTION x(a BIGINT) RETURNS BIGINT RETURN 1
  SESSION a = 1
SELECT * FROM t  
```

Reply:
Changed to `[ WITH [ SESSION name = expression [, ...] ] [ FUNCTION sql_routines ] ]`

Reply:
Did you forget to push a change? I don't see it reflected in the docs above.

Reply:
No, it's fixed in a fixup commit. Let me squash commits.